SMLHelper/Patchers/SpritePatcher.cs

```diff
@@ -1,27 +1,58 @@
 ﻿namespace SMLHelper.V2.Patchers
 {
-    using Assets;
     using System.Collections.Generic;
+    using System.Reflection;
+    using Assets;
 
     internal class SpritePatcher
     {
+        // The groups field is present in Subnautica and BZ Stable.
+        private static readonly FieldInfo groupsInfo = typeof(SpriteManager).GetField("groups", BindingFlags.Static | BindingFlags.NonPublic);
+
+        // In BZ Experimental, it is replaced by the atlases field.
+        private static readonly FieldInfo atlasesInfo = typeof(SpriteManager).GetField("atlases", BindingFlags.Static | BindingFlags.NonPublic);
```

Comment:
Publicized assembly?

Reply:
From the main description
> SpritePatcher now capable of patching Subnautica, Below Zero, and Experimental branches

Yes, this was necessary to make that possible.  
It won't affect in-game performance as this only has to run once during patching.